imports/ui/accounts/accounts-templates.html

```diff
@@ -36,7 +36,7 @@
 
 <template name="override-atTitle">
   <h1 class="title-auth">{{title}}</h1>
-  <p class="subtitle-auth">Signing in allows you to have private lists</p>
+  <p class="subtitle-auth">{{_ 'Signing in allows you to have private lists'}}</p>
```

Comment:
Sounds good to me. I wish there was a good way to generate a blank JSON file with all the keys to hand off to translators?